opencga-app/app/scripts/azure/arm/azurebatch/azuredeploy.json

```diff
@@ -0,0 +1,199 @@
+            "comments": "This storage account is used to associate to a batch account",
+            "sku": {
+                "name": "[parameters('storageAccountsku')]"
+            },
+            "kind": "Storage",
+            "tags": {
+                "ObjectName": "[variables('storageAccountName')]"
+            },
+            "properties": {}
+        },
+        {
+            "type": "Microsoft.Batch/batchAccounts",
+            "name": "[variables('batchAccountName')]",
+            "apiVersion": "2015-12-01",
+            "location": "[parameters('location')]",
+            "tags": {
+                "ObjectName": "[variables('batchAccountName')]"
+            },
+            "properties": {
+                "autoStorage": {
+                    "storageAccountId": "[resourceId('Microsoft.Storage/storageAccounts', variables('storageAccountName'))]"
+                }
+            },
+            "dependsOn": [
+                "[resourceId('Microsoft.Storage/storageAccounts', variables('storageAccountName'))]"
+            ]
+        },
+        {
+            "type": "Microsoft.ContainerInstance/containerGroups",
+            "apiVersion": "2018-10-01",
+            "name": "[substring(concat('batchpool', uniqueString(resourceGroup().id)), 0, 20)]",
+            "location": "[resourceGroup().location]",
+            "dependsOn": [
+              "[resourceId('Microsoft.ManagedIdentity/userAssignedIdentities', variables('identityName'))]",
+              "[resourceId('Microsoft.Authorization/roleAssignments', variables('roleAssignmentIdRg'))]",
+              "[resourceId('Microsoft.Batch/batchAccounts', variables('batchAccountName'))]"
+            ],
+            "identity": {
+              "type": "UserAssigned",
+              "userAssignedIdentities": {
+                "[resourceId('Microsoft.ManagedIdentity/userAssignedIdentities', variables('identityName'))]": {}
+              }
+            },
+            "properties": {
+              "osType": "Linux",
+              "restartPolicy": "Never",
+              "containers": [
+                {
+                  "name": "azure-cli",
+                  "properties": {
+                    "image": "microsoft/azure-cli",
+                    "command": [
+                      "/bin/bash",
+                      "-c",
+                      "[concat('az login --identity -u ', resourceId('Microsoft.ManagedIdentity/userAssignedIdentities', variables('identityName')), ' && az batch account login --name ', variables('batchAccountName'), ' -g ', resourceGroup().name, ' && curl -fsSL ', parameters('_artifactsLocation'), '/azurebatch/configurepool.py', parameters('_artifactsLocationSasToken'), ' > configurepool.py && python3 ./configurepool.py \"', parameters('poolId'), '\" ', parameters('virtualMachineSize'),  ' \"', parameters('mountArgs'),  '\" ', parameters('_artifactsLocation'),  ' ', parameters('_artifactsLocationSasToken'),  ' ', parameters('virtualNetworkSubnetId'))]"
```

Comment:
could the script not login etc. to stop logic being here and in the script?

Reply:
Yup I think that's a good call too, will make this line less horrible to read